code/ui/GameTimer.cs

```diff
@@ -6,27 +6,46 @@ namespace TTTReborn.UI
 {
     public class GameTimer : Panel
     {
-        public Label TimeLabel { set; get; }
 
+        private GameTimerContent gameTimerContent;
         public GameTimer()
         {
             StyleSheet.Load("/ui/GameTimer.scss");
 
-            TimeLabel = Add.Label("00:00", "timelabel");
+            gameTimerContent = new GameTimerContent(this);
         }
 
-        public override void Tick()
+        private class GameTimerContent : Panel
         {
-            // TODO: Handle if Instance is null and if Round is null.
-            if (Game.Instance.Round is Rounds.WaitingRound)
+            public Label TextLabel { set; get; }
+            public Label TimeLabel { set; get; }
+
+            public GameTimerContent(Panel parent)
             {
-                TimeLabel.Text = $"{Game.Instance.Round.RoundName}...";
+                Parent = parent;
 
-                return;
+                TextLabel = Add.Label("asd", "textlabel");
+                TimeLabel = Add.Label("00:00", "timelabel");
             }
+            public override void Tick()
+            {
+                // TODO: Handle if Instance is null and if Round is null.
+                if (Game.Instance.Round is Rounds.WaitingRound)
+                {
+                    TextLabel.Text = $"{Game.Instance.Round.RoundName}...";
+                    TimeLabel.Text = "";
+                    TimeLabel.AddClass("waiting");
```

Comment:
We've used `hide` as a class name for hiding stuff, maybe we should just stick to that.

Reply:
Good point, solved! :)